src/utils/encrypt.ts

```diff
@@ -14,3 +14,14 @@ export const passwordCompare = async (
   const verifyPassword = await bcrypt.compare(inputPassword, userPassword);
   return verifyPassword;
 };
+
```

Comment:
Why did you comment these utility functions?

Reply:
I was using those functions to encrypt and decrypt user data before while verifying the user, but I switched to jwt I was supposed to remove them, but now I have removed them